docs/blog/newsletter-april-2023.rst

```diff
@@ -0,0 +1,93 @@
+- If possible, work with design-first projects that test the API code against the API design. If the API is code-first, there's a risk that each build will overwrite your work.
+- If your contributions are ignored or rejected, try not to take it personally. You're likely to have many of the same challenges you'd have for any other docs work, including maintainers who may be unresponsive or too busy to help you get started.
+- Consider `Google Season of Docs <https://developers.google.com/season-of-docs>`_, which connects interested writers with open-source projects that need documentation support.
+
+------------------------------------
+Measuring productivity of docs teams
+------------------------------------
+
+If your management is looking for measures of documentation productivity, what’s the best way to meet that need?
+
+It’s a tricky problem because many metrics are obvious but unhelpful. Counting team commits is perilously close to bad measures like counting lines of code. Counting words added is almost the opposite of what you want! The number of initiatives launched also doesn’t tell you about their quality.
+
+A bad metric can lead you down the wrong path. Consider Goodhart’s law: “As soon as a measure becomes a target, it ceases to be a good measure.”
+
+One place to start is what you want to achieve, then work backwards to a metric. If productivity means *velocity* to your team: perhaps tickets closed. If it's *solving problems* for your audience: possibly content satisfaction. For providing a *good service* to other teams: maybe backlog size growth. But be careful with what you’re aiming for. For example, a slow turnaround time can be good if it means well-thought out improvements that improve your users’ lives.
+
+It’s worth digging into what management is really looking for. Authoring is not like manufacturing: the visible output (topics published, page views) isn't well correlated with effort. A better angle is impact on the business – such as the diagram at the end of this `article on the value of documentation <https://document360.com/blog/value-of-documentation/>`__. Depending on what your users say, your goal might be increasing time spent on the docs website or reducing support tickets.
+
+Whatever you do, you should think it through. “What gets measured gets done”: if you measure quantity not quality, your docs will suffer.
+
+------------------
+Setting boundaries
+------------------
+
+As a documentarian, it’s common to seek feedback on your work from others, especially people who aren't exclusively focused on documentation.It’s important to remember that not everyone has the same level of expertise or understanding of the writing process. This can lead to reviewers overstepping boundaries or providing feedback that may not be helpful or relevant.
```

Comment:
```suggestion
As a documentarian, it’s common to seek feedback on your work from others, especially people who aren't exclusively focused on documentation. It’s important to remember that not everyone has the same level of expertise or understanding of the writing process. This can lead to reviewers overstepping boundaries or providing feedback that may not be helpful or relevant.
```